datashuttle/datashuttle.py

```diff
@@ -148,7 +148,7 @@ def create_folders(
         ses_names: Optional[Union[str, List[str]]] = None,
         datatype: Union[str, List[str]] = "",
         bypass_validation: bool = False,
-    ) -> List[Path]:
+    ) -> Dict[str, Path]:
```

Comment:
this function is missing a "Returns" docstring section. Perhaps you can add it, describing the contents of the returned dict?